components/brave_wallet_ui/assets/svg-icons/graphic-lock-icon.svg

```diff
@@ -0,0 +1,76 @@
+<svg width="103" height="88" viewBox="0 0 103 88" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you run these graphics through SVGO / https://jakearchibald.github.io/svgomg/ please? Looks like we can reduce size by 60% without any visual effect.